dts/bindings/sensor/ti,ina23x.yaml

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2021 The Chromium OS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: TI INA230 and INA231 Bidirectional Current and Power Monitor
```

Comment:
can you document here how `include/dt-bindings/sensor/ina23x.h` is expected to be used?